lib/vmdb/settings/database_source.rb

```diff
@@ -3,23 +3,51 @@ class Settings
     class DatabaseSource
       include Vmdb::Logging
 
-      attr_reader :resource
+      attr_reader :resource, :settings_change_holder
 
-      def initialize(resource)
-        @resource = resource
+      def initialize(resource_to_get_settings_about, class_to_call_settings_changes_on)
+        @resource = resource_to_get_settings_about
+        @settings_change_holder = class_to_call_settings_changes_on
```

Comment:
You knew I was gonna say something about the naming here 😜 

So, I'm wondering if it's better to doc the params, and then have the param names match the attr_reader names.  Not quite sure.